src/java/org/apache/cassandra/db/ColumnFamilyStore.java

```diff
@@ -2423,7 +2423,7 @@ private AbstractScanIterator getSequentialIterator(final DataRange range, long n
         assert !(range.keyRange() instanceof Range) || !((Range<?>)range.keyRange()).isWrapAround() || range.keyRange().right.isMinimum() : range.keyRange();
 
         final ViewFragment view = select(viewFilter(range.keyRange()));
-        Tracing.trace("Executing seq scan across {} sstables for {}", view.sstables.size(), range.keyRange().getString(metadata.getKeyValidator()));
+        Tracing.trace("Executing seq scan across {} sstables", view.sstables.size());
 
```

Comment:
could we instead wrap this call in `if(Tracing.isTracing())`?

Reply:
We certainly can, nice idea!